printer/print.go

```diff
@@ -145,6 +145,6 @@ func writeImportHeader(b *bytes.Buffer, imports ...string) {
 }
 
 func writeBuildHeader(b *bytes.Buffer, buildHeaders []string) {
-	headers := fmt.Sprintf("// +build %s\n\n", strings.Join(buildHeaders, " "))
+	headers := fmt.Sprintf("//go:build %s\n\n", strings.Join(buildHeaders, " "))
```

Comment:
I wondered if it is worth supporting both for a while? This doesn't match your change to use both headers in https://github.com/algorand/go-algorand/pull/3919 for example

Reply:
Good point...I was worried about the non-backwards compatible nature of this. I agree it's probably best to just add both sets of headers and it's an easy change to make too. Will update.